bin/differential_hash_expression.py

```diff
@@ -282,9 +287,19 @@ def write_hash_coefficients(coefficients, group, threshold):
 
     moltype = calculate_moltype(args)
 
-    main(args.metadata_csv, args.ksize, moltype, args.group_col, args.group1,
-         args.sig_col,
-         args.threshold, args.verbose, args.inverse_regularization_strength,
-         args.solver, args.penalty, args.n_jobs,
-         args.random_state, args.use_sig_basename, args.max_group_size,
-         args.with_abundance)
+    main(metadata_csv=args.metadata_csv,
+         ksize=args.ksize,
+         molecule=moltype,
+         group_col=args.group_col,
+         group1=args.group1,
+         sig_col=args.sig_col,
+         threshold=args.threshold,
+         verbose=args.verbose,
+         C=args.inverse_regularization_strength,
```

Comment:
why is this `C'? capital letters are reserved for constants and might have issues